indico/modules/events/surveys/controllers/management.py

```diff
@@ -266,3 +268,60 @@ def _render_questionnaire(survey):
     tpl = get_template_module('events/surveys/_questionnaire.html')
     form = make_survey_form(survey.questions)()
     return tpl.render_questionnaire(survey, form)
+
+
+class RHExportSubmissions(RHManageSurveyBase):
+    """Export submissions from the survey to a CSV file"""
+
+    def _process(self):
+        if not self.survey.submissions:
+            flash(_('There are no submissions in this survey'))
+            return redirect(url_for('.manage_survey', self.survey))
+
+        form = self._prepare_form()
+        if form.validate_on_submit():
+            csv_file = generate_csv_from_survey(self.survey, set(form.submitters.data))
+            delete_file.apply_async(args=[csv_file.name], countdown=3600)
+            return send_file('submissions-{}.csv'.format(self.survey.id), csv_file.name, 'text/csv')
+        return WPManageSurvey.render_template('submission_export.html', self.event, form=form, survey=self.survey)
```

Comment:
do we want/need a separate page for this? we could simply add checkboxes to the submission list... (nothing selected = everything, otherwise only the selected ones). maybe with some javascript that changes the button title from "Export all submissions" to "Export selected submissions" in case something is selected